deployment/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     image: 'mongo'
     container_name: 'lodestar-mongo'
     environment:
-      - MONGO_INITDB_DATABASE=engagement
+      - MONGO_INITDB_DATABASE=artifacts
```

Comment:
looks like the db name is still `engagement` in the application.properties file so was this name supposed to change? if it was, just disregard.

Reply:
yep . not sure what happened here.